### DISCUSSION TITLE
Discussion page : cannot "View more" ?

### DISCUSSION THREAD

**Discussion by pcar75** [community/community#31174] [Discussions]
Bonjour,
there seems to be a problem with the "View more" link of a comment in a Discussion page :
https://github.com/jamulussoftware/jamulus/discussions/2807
One of my comment says 6 replies and displays 5, When I click the "View more" link to see the 6th reply, all replies disappear and the main comment displays 0 reply.

---------Before---------------
![GitHub_Jamulus Discussion_01](https://user-images.githubusercontent.com/64435162/187108158-3e8c89ae-c864-4c1c-be69-4dec40a76d88.PNG)

----------After-----------------
![GitHub_Jamulus Discussion_02](https://user-images.githubusercontent.com/64435162/187108169-182574c1-908a-45fd-b3be-93d05d4a733c.PNG)

I never experienced this before and I remember reading Topics having comments with their "View more" displaying a lot more and longer replies, logged in or not.

I use Firefox 104.0 in Windows 10 :

When logged in, in full sized window , a top button appears "Show 1 previous reply" ; functions OK.

When logged in, in partial sized window , a top link appears "Show 1 previous reply" ; functions OK.

When not logged in, in full sized window , a bottom link appears "View More" ; functions strips replies and status displays 0 replies

When not logged in, in partial sized window , a bottom link appears "View More" ; functions strips replies and status displays 0 replies

Is this behavior normal ?!?!

**Comment by queenofcorgis** [comment]
Hi there - thanks for all the details, I've moved this to the appropriate category and triaged it will labels so the proper teams can have visibility.
